### PR TITLE
feat(Helper): add component, demo, docs

### PR DESCRIPTION
**Pull request by andretchen0**
Ports Drei's `<Helper />` to Cientos. Adds:

* component
* demo
* docs

### CODE REVIEW

File: src/core/controls/Helper/useHelper.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+type HelperType = Object3D & { update: () => void, dispose: () => void }
+type HelperConstructor = new (...args: any[]) => any
+
+export function useHelper<T extends HelperConstructor>(
+  object3D: MaybeRefOrGetter<Object3D | null | undefined | false>,
+  helperConstructor: T,
+  ...args: any[]
+) {
+  const helper = shallowRef<HelperType>()
+  const scene = useTres().scene
```
Comment (alvarosabu) [inline src/core/controls/Helper/useHelper.ts:18]:
```suggestion
  const { scene } = useTres()
```

File: src/core/controls/Helper/useHelper.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  let currentHelper: HelperType = undefined!
+
+  watchEffect(() => {
+    if (object3D && toValue(object3D) && helperConstructor) {
+      helper.value = currentHelper = new (helperConstructor as any)(toValue(object3D), ...args)
```
Comment (alvarosabu) [inline src/core/controls/Helper/useHelper.ts:24]: @andretchen0 any chance we can avoid using `any` assertion here?

**Review by alvarosabu — CHANGES_REQUESTED** [review]

File: src/core/controls/Helper/useHelper.ts (hunk 2 above)
Comment (andretchen0) [reply src/core/controls/Helper/useHelper.ts]: No problem. I made the change.

For what it's worth, the effect of the change is purely cosmetic. The resulting type of `helper.value` is still

```ts
Ref<HelperType | undefined, HelperType | undefined>.value: HelperType | undefined
```

File: src/core/controls/Helper/useHelper.ts (hunk 1 above)
Comment (andretchen0) [reply src/core/controls/Helper/useHelper.ts]: Done

**Pull request merged**